src/deprecated.jl

```diff
@@ -0,0 +1,31 @@
+doi(x::Record) = x.location.doi
+function doi(x::RecordDatabase) 
+  dois = []
+     for i in keys(x)
+         push!(dois, location(get(x.records, i, false)).doi)
+     end
+  return dois
+end
+
+@deprecate doi(x) url(x)
```

Comment:
Do we really need to deprecate this? It probably was used only internally?

Reply:
I agree that it was only used internally. However, it was an exported function and thus formally requires a depricate.